buildenv/jenkins/clang_format/pullInfra.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -f
+###############################################################################
+# Copyright IBM Corp. and others 2025
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+###############################################################################
+
+wget https://raw.githubusercontent.com/eclipse-omr/omr/refs/heads/master/compiler/.clang-format runtime/compiler/
+wget https://raw.githubusercontent.com/eclipse-omr/omr/refs/heads/master/buildenv/jenkins/clang_format/Dockerfile buildenv/jenkins/clang_format/
+wget https://raw.githubusercontent.com/eclipse-omr/omr/refs/heads/master/buildenv/jenkins/clang_format/clang-format-wrapper.sh buildenv/jenkins/clang_format/
```

Comment:
```suggestion
wget https://raw.githubusercontent.com/eclipse-omr/omr/refs/heads/master/compiler/.clang-format -P runtime/compiler/
wget https://raw.githubusercontent.com/eclipse-omr/omr/refs/heads/master/buildenv/jenkins/clang_format/Dockerfile -P buildenv/jenkins/clang_format/
wget https://raw.githubusercontent.com/eclipse-omr/omr/refs/heads/master/buildenv/jenkins/clang_format/clang-format-wrapper.sh -P buildenv/jenkins/clang_format/
```
Missed the `-P` flag.